test/metabase/public_settings/premium_features_test.clj

```diff
@@ -210,7 +210,7 @@
   "Returns a greeting for a user. Only EE version is defined with defenterprise-schema."
   metabase-enterprise.util-test
   [username]
-  (format "Hi %s, you're an OSS customer!"))
+  (format "Hi %s, you're an OSS customer!" username))
```

Comment:
This was another error that was caught by switching `defenterprise` to a custom hook.